cli/src/pcluster/utils.py

```diff
@@ -141,11 +141,21 @@ def generate_random_name_with_prefix(name_prefix):
 
     Example: <name_prefix>-4htvo26lchkqeho1
     """
-    random_string = "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(16))
+    random_string = generate_random_prefix()
     output_name = "-".join([name_prefix.lower()[: 63 - len(random_string) - 1], random_string])
     return output_name
 
 
+def generate_random_prefix():
+    """
+    Generate a random prefix that is 16 characters long.
+
+    Example: 4htvo26lchkqeho1
+    """
+    random_string = "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(16))
```

Comment:
minor you can avoid the defintion of the internal random_string variable and just have the return call.

Reply:
Done